src/service/client.rs

```diff
@@ -11,6 +11,7 @@ use futures::channel::mpsc::{self, Sender};
 use futures::future::BoxFuture;
 use futures::sink::SinkExt;
 use lsp_types::*;
+use lsp_types::request::WorkspaceDiagnosticRefresh;
```

Comment:
Just a stranger, but should it be placed at line 350 instead of here for consistency?